.github/workflows/pull-request.yaml

```diff
@@ -50,7 +50,8 @@ jobs:
           mkdir $HOME/.snowsql
           envsubst < deploy/opscenter.config > $HOME/.snowsql/config
       - name: Run deploy/devdeploy.py (without triggering materializations)
-        run: poetry run python deploy/devdeploy.py --profile opscenter --skip-finish-setup
+        # Some unit tests call procedures which expect admin.finalize_setup() to have been run.
```

Comment:
doesn't this materialize all the data too? Expensive isn't int?

Reply:
not sure an alternative, just run `finalize_setup` w/o all the other stuff?

Reply:
can we run this in the cypress job so we re-use the db?

Reply:
> not sure an alternative, just run finalize_setup w/o all the other stuff?

Not an alternative, I think, as the tasks would be manually triggered at the end of `finalize_setup()`

> can we run this in the cypress job so we re-use the db?

That makes sense to me. We can do one, real deploy (with data), and run many tests against it.